packages/react-components/react-motion-components-preview/stories/src/Rotate/RotateCardFlip.stories.tsx

```diff
@@ -0,0 +1,200 @@
+    exitDuration: motionTokens.durationUltraSlow,
+  },
+  {
+    id: 'spin',
+    name: 'Spin',
+    description: 'Z-axis rotation',
+    icon: '🔄',
+    color: tokens.colorPaletteRedForeground2,
+    axis: 'z',
+    angle: 180,
+    easing: curveSpringRelaxed,
+    exitEasing: motionTokens.curveDecelerateMid,
+    duration: motionTokens.durationUltraSlow * 4, // 2000ms = 500ms * 4
+    exitDuration: motionTokens.durationUltraSlow,
+  },
+];
+
+export const CardFlip = () => {
+  const classes = useClasses();
+  const [activePatterns, setActivePatterns] = React.useState<Set<string>>(new Set(patterns.map(p => p.id)));
+
+  const togglePattern = (patternId: string) => {
+    setActivePatterns(prev => {
+      const newSet = new Set(prev);
+      if (newSet.has(patternId)) {
+        newSet.delete(patternId);
+      } else {
+        newSet.add(patternId);
+      }
+      return newSet;
+    });
+  };
+
+  const toggleAllPatterns = () => {
+    if (activePatterns.size === patterns.length) {
+      // All are showing, so hide all
+      setActivePatterns(new Set());
+    } else {
+      // Some or none are showing, so show all
+      setActivePatterns(new Set(patterns.map(p => p.id)));
+    }
+  };
+
+  const getToggleButtonText = () => {
+    return activePatterns.size === patterns.length ? 'Flip to Back' : 'Flip to Front';
+  };
+
+  return (
+    <div className={classes.container}>
+      <div className={classes.controls}>
+        <Button onClick={toggleAllPatterns} aria-label="Toggle all rotation patterns">
+          {getToggleButtonText()}
+        </Button>
+      </div>
+
+      <div className={classes.patternsGrid}>
+        {patterns.map(pattern => (
+          <div key={pattern.id} className={classes.cardWrapper}>
+            <Rotate
+              visible={activePatterns.has(pattern.id)}
+              axis={pattern.axis}
+              angle={pattern.angle}
+              duration={pattern.duration}
+              easing={pattern.easing}
+              exitEasing={pattern.exitEasing}
+              exitDuration={pattern.exitDuration}
+              animateOpacity={false}
+            >
+              <CompoundButton
+                appearance="outline"
+                className={classes.patternCard}
+                onClick={() => togglePattern(pattern.id)}
+                icon={
+                  <div className={classes.demoIcon} style={{ backgroundColor: pattern.color }}>
+                    {pattern.icon}
+                  </div>
+                }
+                secondaryContent={pattern.description}
+                aria-label={`Toggle ${pattern.name} rotation`}
+                onKeyDown={(e: React.KeyboardEvent) => {
+                  if (e.key === 'Enter' || e.key === ' ') {
+                    e.preventDefault();
+                    togglePattern(pattern.id);
+                  }
+                }}
+              >
+                {pattern.name}
+              </CompoundButton>
```

Comment:
```suggestion
              <CompoundButton
                appearance="outline"
                aria-label={`Toggle ${pattern.name} rotation`}
                aria-pressed={activePatterns.has(pattern.id)}
                className={classes.patternCard}
                icon={
                  <div className={classes.demoIcon} style={{ backgroundColor: pattern.color }}>
                    {pattern.icon}
                  </div>
                }
                onClick={() => togglePattern(pattern.id)}
                secondaryContent={pattern.description}
              >
                {pattern.name}
              </CompoundButton>
```

`onKeyDown` does not seem to be needed as it's a button now, let's also add `aria-pressed` like `TooggleButton` has